### PR TITLE
fix tarball path in github actions

### PR DESCRIPTION
**Pull request by bazsi**
This branch fixes the workaround in #3486 properly.

This was tested as a part of a larger pile of release-automation tests here: https://github.com/syslog-ng/syslog-ng/runs/1464722837?check_suite_focus=true

It also adds a new variable in draft-release.yml which helped me to understand that piece of code better.

### CODE REVIEW

**Comment by kira-syslogng** [comment]
Build SUCCESS

**Comment by bazsi** [comment]
As mentioned by @gaborznagy in https://github.com/syslog-ng/syslog-ng/pull/3464#discussion_r531599055

I have added the removal of a tarball backup copy (pkg-tarball) and also ensured that no extra files get added into the tarball by
doing a sweep first.

**Comment by kira-syslogng** [comment]
Build SUCCESS

File: dbld/functions.sh [hunk 1]
```diff
@@ -153,9 +153,13 @@ function validate_container() {
 function capture_artifacts()
 {
 	if [ "$MODE" = "release" ]; then
+		ARTIFACT_DIR="/dbld/release/$VERSION"
+		echo "Capturing artifacts:" "$*" "into" "${ARTIFACT_DIR}"
 		for artifact in "$*"; do
-			cp ${artifact} /dbld/release/${VERSION}
+			cp ${artifact} "${ARTIFACT_DIR}"
 		done
```
[the remark this answers is not in the document]
Comment (bazsi) [reply dbld/functions.sh]: done.

**Comment by bazsi** [comment]
Rebased and comment resolved.

**Review by szemere — APPROVED** [review]

**Comment by kira-syslogng** [comment]
Build SUCCESS

**Review by MrAnno — APPROVED** [review]

**Pull request merged**